include/seqan3/argument_parser/validators.hpp

```diff
@@ -264,15 +264,25 @@ class value_list_validator
  */
 //!\brief Deduction guide for a parameter pack over an arithmetic type.
 template <arithmetic ...option_types>
+//!\cond
+    requires !(std::same_as<char, option_types> || ...)
+//!\endcond
 value_list_validator(option_types...) -> value_list_validator<double>;
 
 //!\brief Deduction guide for ranges over an arithmetic type.
 template <std::ranges::forward_range range_type>
 //!\cond
-    requires arithmetic<std::ranges::range_value_t<range_type>>
+    requires arithmetic<std::ranges::range_value_t<range_type>> &&
+             !std::same_as<std::ranges::range_value_t<range_type>, char>
 //!\endcond
 value_list_validator(range_type && rng) -> value_list_validator<double>;
 
+//!\brief C-style string deduces to std::string.
+value_list_validator(char const *) -> value_list_validator<std::string>;
+
+//!\brief C-style string deduces to std::string.
+value_list_validator(char *) -> value_list_validator<std::string>;
```

Comment:
why was the one below not picked up?

Reply:
I don't know? Because parameter packs cannot capture a char pointer into a pack of chars?